[1.2.3] main 함수에서의 충돌 함수들 호출

hongyj9823 opened the pull request:
1. main 함수에서 기존의 경계충돌 조건문 삭제 및 경계충돌 함수(isCollideBoundary) 호출
2. main 함수에서 자가충돌 함수(isCollideBody) 호출
3. main 함수에서의 기존의 과일충돌 조건문 삭제 및 과일충돌 함수(isCollideFruit) 호출
4. 변수 quit 하나만 사용해서 나타나는 오류 해결(조건문 추가)
5. 맵의 경계 충돌에 사용되는 범위 수정

hongyj9823 committed:
[MOD] Delete former statement and add new function

I deleted the former conditional statements of collision between the snake's head and boundaries. I called the new function in main file.

hongyj9823 committed:
[MOD] Call the isCollideFruit function in main file

I deleted the former conditional statements of collision between fruit and the snake's head and call the isCollideFruit function in main file.

hongyj9823 committed:
[NEW] Call the isCollideBody function in main file

I called the isCollideBody function in main file.

SonHyeWon commented:
함수는 맵 범위에 맞게 약간의 좌표 수정이 필요해 보이고, main에서 quit 변수에 값을 주는 방식이 개선이 필요해보입니다.. do-while 조건문에는 두번째로 값을 받는 quit의 값만 들어가는 듯 하네요. 확인 부탁드립니다!

hongyj9823 commented:
오류 확인했습니다! 개선 후 다시 브랜치 올리도록 하겠습니다. 감사합니다.

hongyj9823 committed:
[BUG] Solve the problem that use one quit

I added the conditional statement to isCollideBody function.I can use one word to quit program.

hongyj9823 committed:
[MOD] Modify the range of the isCollideBoundary

When I ran the program, the boundaries are incorrect. So I modified the range.

mellime commented:
최근 머지 내역만 확인 부탁드립니다. 코드 추가, 수정 부분 확인했습니다.

hongyj9823 commented:
네.알겠습니다. 감사합니다.
